src/test/java/org/jitsi/meet/test/pageobjects/web/ToolbarV2.java

```diff
@@ -0,0 +1,416 @@
+        clickButtonInOverflowMenu(SHARE_VIDEO);
+    }
+
+    /**
+     * Waits up to 10 seconds for the video mute button in the toolbar to be
+     * visible.
+     */
+    public void waitForVideoMuteButtonDisplay()
+    {
+        waitForButtonDisplay(VIDEO_MUTE);
+    }
+
+    /**
+     * Waits up to 3 seconds for the toolbar to be visible.
+     */
+    public void waitForVisible()
+    {
+        TestUtils.waitForDisplayedElementByID(
+            participant.getDriver(),
+            TOOLBOX_ID,
+            3);
+    }
+
+    /**
+     * Try to click on a button but do not fail if it is not clickable.
+     *
+     * @param accessibilityLabel The accessibility label of the button to be
+     * clicked.
+     */
+    private void attemptToClickButton(String accessibilityLabel)
+    {
+        MeetUIUtils.clickOnElement(
+            participant.getDriver(),
+            getAccessibilityCSSSelector(accessibilityLabel),
+            false
+        );
+    }
+
+    /**
+     * Try to click on a button and fail if it cannot be clicked.
+     *
+     * @param accessibilityLabel The accessibility label of the button to be
+     * clicked.
+     */
+    private void clickButton(String accessibilityLabel)
+    {
+        MeetUIUtils.clickOnElement(
+            participant.getDriver(),
+            getAccessibilityCSSSelector(accessibilityLabel),
+            true
+        );
+    }
+
+    /**
+     * Ensure the overflow menu is open and clicks on a specified button.
+     *
+     * @param accessibilityLabel The accessibility label of the button to be
+     * clicked.
+     */
+    private void clickButtonInOverflowMenu(String accessibilityLabel)
+    {
+        openOverflowMenu();
+
+        clickButton(accessibilityLabel);
+    }
+
+    /**
+     * Helper for formatting the string to be used as a CSS selector for
+     * an accessibility label.
+     *
+     * @param accessibilityLabel The accessibility label to be used to search
+     * for a WebElement on the page.
+     * @return String intended to be used with By#cssSelector.
+     */
+    private String getAccessibilityCSSSelector(String accessibilityLabel)
+    {
+        return String.format("[aria-label=\"%s\"]", accessibilityLabel);
+    }
+
+    /**
+     * Helper for retrieving the selector for the overflow menu.
+     *
+     * @return <tt>By</tt> to be used by WebDriver to locate the overflow menu.
+     */
+    private By getOverflowMenuSelector()
+    {
+        return By.cssSelector(getAccessibilityCSSSelector(OVERFLOW_MENU));
+    }
+
+    /**
+     * Waits up to 10 seconds for a button to be visible on the toolbar.
+     *
+     * @param accessibilityLabel The accessibility label of the button to be
+     * clicked.
+     */
+    private void waitForButtonDisplay(String accessibilityLabel)
+    {
+        TestUtils.waitForElementBy(
+            participant.getDriver(),
+            By.cssSelector(getAccessibilityCSSSelector(accessibilityLabel)),
+            10);
+    }
+}
```

Comment:
nice helpers in here, keeps a lot of much out of the way when you're reading the methods.